exp/controllers/gcpmanagedcluster_controller.go

```diff
@@ -225,6 +228,11 @@ func (r *GCPManagedClusterReconciler) reconcileDelete(ctx context.Context, clust
 	log := log.FromContext(ctx)
 	log.Info("Reconciling Delete GCPManagedCluster")
 
+	if clusterScope.GCPManagedControlPlane != nil {
+		log.Info("GCPManagedControlPlane not deleted yet, retry later")
+		return ctrl.Result{RequeueAfter: 5 * time.Minute}, nil
```

Comment:
Same comments as above with the requeue time.

Reply:
done